_inc/client/components/jetpack-connect/style.scss

```diff
@@ -3,7 +3,7 @@
 
 	// modding header__label specifically on the connection page.
 	.dops-section-header__label {
-		margin: 1px; // allows box-shadow of parent to display properly.
+		margin: rem( 1px) ; // allows box-shadow of parent to display properly.
```

Comment:
Not sure anything that's 1px outside of line spacing really needs to be rem-idied.